package.json

```diff
@@ -38,10 +38,11 @@
   "scripts": {
     "debug": "node debug app.js",
     "start": "node app.js",
-    "test": "NODE_ENV=test mocha test/bootstrap.test.js test/integration/**/*.test.js"
+    "test": "NODE_ENV=test ./node_modules/.bin/mocha test/bootstrap.test.js test/integration/**/*.test.js"
```

Comment:
Ah true, I'm gonna revert this back then. This ended up coming in as a thing I was trying when my local tests weren't working. 😬